source/Monster.cpp

```diff
@@ -3,68 +3,73 @@
 namespace dungeon {
 
 Slime::Slime() {
-  type = Class::none;
+  type = ClassType::NONE;
   name = "Slime";
-  hpBase = 10000;
-  atkBase = 50;
-  critRateBase = 5;
-  critDamageBase = 50;
-  energy = 5;
-  curHp = hp();
+
+  attr.baseHp = 10000;
+  attr.baseAtk = 50;
```

Comment:
ditto, you can reuse codes. In fact these assignments can be put in Entity's constructor and share with Character.